src/api/app/models/bs_request_action_change_devel.rb

```diff
@@ -24,7 +24,7 @@ def execute_accept(_opts)
     target_package.develpackage = Package.get_by_project_and_name(self.source_project, self.source_package)
 
     target_package.resolve_devel_package
-    target_package.store(comment: "change_devel request #{self.bs_request.id}", requestid: self.bs_request.id)
+    target_package.store(comment: "change_devel request #{self.bs_request.number}", requestid: self.bs_request.number)
```

Comment:
"requestid: self.bs_request.number"

@adrianschroeter That's intended? Or should we keep using the id as identifier?